src/styles/bloks/catalog.scss

```diff
@@ -0,0 +1,123 @@
+.main {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 9th point